docs/installation.rst

```diff
@@ -82,55 +73,27 @@ Installing
     python setup.py install
 
 
-Downloading Required Data
-*************************
+Downloading Required Reference Data
+************************************
 
-Pandeia and WebbPSF need the reference datasets.
+STIPS, Pandeia, and WebbPSF need the reference datasets.
 You will need to download the data and add them to your environmental path
 
-1. Check if wget is installed by running::
-
-    wget
-
-    # If it is not installed (usually on Mac OS),
-    # it can be downloaded using Conda
-
-    conda install wget
-
-2. `cd` into a directory you would like to store the data in and run the following commands::
+1. Add the following paths to your bash environmental path. It is recommended that you add the path to your `.bash_profile` file::
 
-    # Synphot and STSynphot reference data
-    wget -qO- https://ssb.stsci.edu/trds/tarfiles/synphot1.tar.gz | tar xvz
-    wget -qO- https://ssb.stsci.edu/trds/tarfiles/synphot2.tar.gz | tar xvz
-    wget -qO- https://ssb.stsci.edu/trds/tarfiles/synphot5.tar.gz | tar xvz
-    wget -qO- https://ssb.stsci.edu/trds/tarfiles/synphot6.tar.gz | tar xvz
+		export stips_data="<absolute_path_to_this_folder>/ref_data/stips_data"
+		export WEBBPSF_PATH="<absolute_path_to_this_folder>/ref_data/webbpsf-data"
+		export PYSYN_CDBS="<absolute_path_to_this_folder>/ref_data/grp/redcat/trds"
+		export pandeia_refdata="<absolute_path_to_this_folder>/ref_data/pandeia_data-x.x.x_roman"
 
+Make sure that you have the correct version number for `pandeia_refdata` (replace the "x.x.x").
 
-    # Pandeia reference data
-    wget -qO- https://stsci.box.com/shared/static/5j506xzg9tem2l7ymaqzwqtxne7ts3sr.gz | tar -xvz
-
-    # WebbPSF reference data
-    wget -qO- https://stsci.box.com/shared/static/qcptcokkbx7fgi3c00w2732yezkxzb99.gz | tar xvz
+2. `cd` into the "ref_data" directory in your STScI-STIPS clone
```

Comment:
```suggestion
2. ``cd`` into the "ref_data" directory in your ``STScI-STIPS`` clone
```
Monospacing STScI-STIPS here matters since it's a reference to the GitHub repository name and not just the package in general.